16/feoh/randomtitlepicker.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import random
+# You'll need to create a file named config.py containing a line like: apikey="<your_api_key>"
+from config import apikey
+
+# Random OMDB title picker.
+
+
+def generate_random_imdb_id():
+    # This is the highest IMDB title id at time of writing. This stinks. Wish I had a better way to
+    # dynamically set this!
+    max_imdb_title_number = 9916754
+    random.seed()
+    random_id = int(random.random() * max_imdb_title_number)
+    return f"tt{random_id}"
+
+
+def make_omdb_request(imdb_title_id):
+    movie_params = {'apikey': apikey, 'i': imdb_title_id}
```

Comment:
lowercase apikey it did not occur to me that it would come from config, use a constant here?